client/src/BooksList/BooksList.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+//import data from './books.json';
+import './BooksList.css';
+import Rater from 'react-rater';
+import 'react-rater/lib/react-rater.css';
+
+const axios = require('axios');
+
+class BooksList extends Component {
+
+    axios.get('/books')
```

Comment:
Does it works ? It should be inside constructor or some function. I have error during compilation

Reply:
Also tests fail because of it